ansible/roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -1078,6 +1090,20 @@ g_template_openshift_master:
     - "Openshift Master process not running on {HOST.NAME}"
     priority: avg
 
+  - name: "Route returned non-200 HTTP status on {HOST.NAME}"
```

Comment:
Thinking this should be standardised messaging as per the above: 
"App Create returned non-200 HTTP status on ROUTE"